### PR TITLE
update mAP analysis

### PR DESCRIPTION
**Pull request by MikeLippincott**
This PR focuses on performing mAP analysis at the individual perturbation level and visualizing the results

### CODE REVIEW

File: 9.mAP/notebooks/4.generate_map_scores_morphology_treatment.ipynb [hunk 1]
```diff
@@ -0,0 +1,1197 @@
+{
```
Comment (axiomcura) [inline 9.mAP/notebooks/4.generate_map_scores_morphology_treatment.ipynb:1]: <p>Consider removing the print statements that are commented out!</p><p> </p><p>Another thing that I am noticing is the total number of positive pairs. It's really small. Is this consistent throughout the analysis? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

File: 9.mAP/notebooks/5.aggregate_map_scores_morphology_treatment.ipynb [hunk 2]
```diff
@@ -0,0 +1,971 @@
+{
```
Comment (axiomcura) [inline 9.mAP/notebooks/5.aggregate_map_scores_morphology_treatment.ipynb:1]: <p>where there any that passed the threshold? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/123/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

**Review by axiomcura — APPROVED** [review]
LGTM! Nice to see that you are using mAP to a whole another level! 

I had little trouble understanding why you were comparing between morphology and secretome mAP scores. If you have a chance, mind giving me a small explanation. Then I can for sure be more helpful trying to review your plots !

File: 9.mAP/notebooks/4.generate_map_scores_morphology_treatment.ipynb (hunk 1 above)
Comment (MikeLippincott) [reply 9.mAP/notebooks/4.generate_map_scores_morphology_treatment.ipynb:1]: there are only 4-8 well-level replicates for each treatment so yes. The cell death grouped treatments have a more robust sample size

**Pull request merged**